addons/rook/template/base/install.sh

```diff
@@ -143,13 +148,6 @@ function rook_cluster_deploy() {
     local src="${DIR}/addons/rook/${ROOK_VERSION}/cluster"
     local dst="${DIR}/kustomize/rook/cluster"
 
-    # Don't redeploy cluster - ekco may have made changes based on num of nodes in cluster
-    if kubectl -n rook-ceph get cephcluster rook-ceph >/dev/null 2>&1 ; then
-        echo "Cluster rook-ceph already deployed"
-        rook_cluster_deploy_upgrade
-        return 0
-    fi
-
```

Comment:
Are you sure this won't be undoing changes made by ekco?
Or are we just fine with those?